compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1931,3 +1931,7 @@ error.duplicate.flags=\
 
 error.cannot.import.module.generated.for.a.client.decl=\
   a module generated for a client declaration cannot be imported
+
+error.outer.join.must.be.declared.with.var=\
+  outer join must be declared with var
```

```suggestion
  outer join must be declared with ''var''
```

---

one quote is enough right? :thinking: 
```suggestion
outer join must be declared with 'var'
```

---

One will be skipped https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/resources/compiler.properties#L1289